resources/schema.py

```diff
@@ -590,7 +597,10 @@ class Query:
     piers = DjangoFilterConnectionField(PierNode)
 
     harbor = relay.Node.Field(HarborNode)
-    harbors = DjangoFilterConnectionField(HarborNode)
+    harbor_by_servicemap_id = graphene.Field(
+        HarborNode, servicemap_id=graphene.String(required=True)
+    )
+    harbors = DjangoFilterConnectionField(HarborNode, filterset_class=HarborFilter)
```

Comment:
does it still allow using `filter_fields` defined on `HarborNode`?

Reply:
Now it does. 😄